chef/cookbooks/glance/recipes/ceph.rb

```diff
@@ -24,35 +27,43 @@
 else
   # If external Ceph cluster will be used,
   # we need install ceph client packages
-  if node[:platform] == "suse" && File.exists?("/etc/ceph/ceph.client.admin.keyring")
+  return unless File.exists?(ceph_conf)
+
+  if node[:platform] == "suse"
     package "ceph-common"
     package "python-ceph"
```

Comment:
BTW, who is responsible for installing ```python-ceph``` when using a crowbar deployed ceph cluster?

AFAICS, ```ceph::default```, which is included by ```ceph::keyring``` only installs the ceph-common package. Or is python-ceph not required in that case?